scss/utilities/_position.scss

```diff
@@ -23,3 +23,8 @@
     z-index: $zindex-sticky;
   }
 }
+
+.pos-s { position: static; }
```

Comment:
Because there's no risk of overlap with padding—these values are unique to positioning.